src/client/polyfills/css-shim/test/css-shim.spec.ts

```diff
@@ -366,8 +366,8 @@ describe('css-shim', () => {
     );
   });
 
-  var window: Window;
-  var document: Document;
+  var window: Window; // eslint-disable-line no-var
+  var document: Document; // eslint-disable-line no-var
```

Comment:
One last thing - can we add the justification for disabling eslint here?
```suggestion
  var window: Window; // eslint-disable-line no-var -- shims will continue to use var while we support older browsers
  var document: Document; // eslint-disable-line no-var -- shims will continue to use var while we support older browsers
```